src/styles/main.css

```diff
@@ -0,0 +1,90 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.calendar {
+  counter-reset: day-number 0;
+  display: flex;
+  flex-wrap: wrap;
+  width: calc(100px * 7 + 1px * 6 + 10px * 2);
+  padding: 10px;
+  gap: 1px;
+}
+
+.calendar__day {
+  width: 100px;
+  height: 100px;
+  background-color: #eee;
```

Comment:
You are using a color code directly in your CSS. Consider using a SASS variable for this color, as it will make it easier to maintain and change in the future.